ocean_lib/services/test/test_service.py

```diff
@@ -155,10 +154,10 @@ def test_utilitary_functions_for_trusted_algorithm_publishers(publisher_ocean):
     compute_service = ddo.services[1]
     assert compute_service.type == "compute"
 
-    addr1 = accounts.add().address
+    addr1 = publisher_ocean.config_dict["web3_instance"].eth.account.create().address
```

Comment:
`publisher_ocean.config_dict["web3_instance"]` is repeated 3 times in a row. Hard to grok, and verbose.

I recommend:
```python
web3 = publisher_ocean.config_dict["web3_instance"]
addr1 = web3.eth.account.create().address() 
..
addr2 = web3.eth.account.create().address() 
..
addr2 = web3.eth.account.create().address() 
```

Reply:
fixed